state/executor.go

```diff
@@ -241,6 +238,27 @@ func (e *BlockExecutor) ApplyBlock(ctx context.Context, state types.State, block
 	return state, resp, nil
 }
 
+// ExtendVote calls the ExtendVote ABCI method on the proxy app.
+func (e *BlockExecutor) ExtendVote(ctx context.Context, block *types.Block) ([]byte, error) {
+	resp, err := e.proxyApp.ExtendVote(ctx, &abci.RequestExtendVote{
+		Hash:   block.Hash(),
+		Height: int64(block.Height()),
+		Time:   block.Time(),
+		Txs:    block.Data.Txs.ToSliceOfBytes(),
+		ProposedLastCommit: abci.CommitInfo{
+			Round: 0,
```

Comment:
instead of `0` and `nil` should you refill using the last extended commit?

Reply:
Good catch - this shouldn't be empty. On the other hand it's hard to fill those fields because there is no voting in rollkit ;)